resources/platformsh/common/3.3.x-dev/.platform/services.yaml

```diff
@@ -76,7 +76,7 @@ rediscache:
 #    type: elasticsearch:7.7
 #    disk: 512
 
-# Due to logic in app/config/env/platformsh.php, do not change the service name to something different than 'varnish'
+# Due to logic in vendor/ezsystems/ezplatform-core/src/EzPlatformCoreBundle/bundle/DependencyInjection/EzPlatformCoreExtension.php, do not change the service name to something different from 'varnish'
```

Comment:
Could you please use FQCN instead of file path ?

Reply:
Good idea. Fixed in aeabd158941e8fb25379a18593e1fed15321fcfc